docs/content/docs/advanced/offline-browsing.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Offline browsing
+incomplete: true
+---
+
+> We live in a disconnected & battery powered world,
+> but our technology and best practices are a leftover
+> from the always connected & steadily powered past.
+>
+> *from [OfflineFirst.org](http://offlinefirst.org/)
+
+Imagine that users visit your website for the first time,
+all the data will be saved into their device storage and they can view it
+on the run without a internet connection.
+
+That's awesome right ? And you know what ?
+Statinamic supports offline browsing **out of the box.**
+
+Here is two different technique to enable offline experience for your website:
+
+## AppCache
+
+This is the old one, IE 10 (and above) supports it.
+
+> Learn about AppCache here: [http://www.html5rocks.com/en/tutorials/appcache/beginner/]
+> [CanIUse -- AppCache](http://caniuse.com/#search=appcache)
+
+You can config Statinamic to generate a AppCache manifest for you after build.
+
+To enable this feature, add an ``appcache`` field to ``package.json`` under
+``statinamic`` section.
+
+```js
+{
+  ...
+  "statinamic": {
+    "appcache": "**/*.*"
```

Comment:
Only match html and json ? 

`**/*.+(html|json)` . glob uses minimatch under the hood